test/Sensors/Sentinel1/Sentinel1MetadataTest.jl

```diff
@@ -149,23 +150,21 @@ end
 
 function BurstTest()
     #Action
-    metaDict = SARProcessing.getDictofXml(SENTINEL1_SLC_METADATA_TEST_FILE)
-    burstinfo = SARProcessing.BurstsInfo(metaDict);
+    meta_dict = SARProcessing.read_xml_as_dict(SENTINEL1_SLC_METADATA_TEST_FILE)
+    bursts = SARProcessing.get_sentinel1_burst_information(meta_dict);
 
     ## Assert
-    check = burstinfo.numberOfBurst == 9
-    check &= length(burstinfo.bursts) == 9
-    check &= isapprox(burstinfo.bursts[1].dopplerCentroid.dcT0,0.00534423320003329; atol = 0.000000001)
-    check &= isapprox(burstinfo.bursts[8].dopplerCentroid.dcT0,0.005342927742124565; atol = 0.000000001)
-    check &= length(burstinfo.bursts[1].azimuthFmRates.azimuthFmRatePolynomial) == 3
-    check &= burstinfo.bursts[5].dopplerCentroid.burstMidTime == Millisecond(12588)
-    check &= burstinfo.bursts[3].sensingTime == DateTime(2022,09,18,07,49,28,166)   
-    check &= isapprox(burstinfo.bursts[2].azimuthFmRates.azimuthFmRateT0 ,0.006018535512387027; atol = 0.000001) 
+    check = length(bursts) == 9
+    check &= isapprox(bursts[1].doppler_centroid.t0,0.00534423320003329; atol = 0.000000001)
+    check &= isapprox(bursts[8].doppler_centroid.t0,0.005342927742124565; atol = 0.000000001)
+    check &= length(bursts[1].azimuth_fm_rate.polynomial) == 3
+    check &= bursts[3].sensing_time == DateTime(2022,09,18,07,49,28,166)   
+    check &= isapprox(bursts[2].azimuth_fm_rate.t0 ,0.006018535512387027; atol = 0.000001) 
```

Comment:
just a small idea, maybe we could add a small check to see that bursts are ordered correctly? Don't know if this could ever happen, but just a thought. Something like:
`check &= all([burst[i].burst_time < burst[i+1].burst_time for i in 1:length(burst)-1])`

Reply:
maybe safer with mid_burst_time